schemas/context/weblink.schema.json

```diff
@@ -9,7 +9,8 @@
   "$schema": "http://json-schema.org/draft-06/schema#",
   "title": "Web Link",
   "type": "object",
-  "description": "Information about the web link of the ExperienceEvent.",
+  "description":
+    "Details about the web link (URL) that corresponds to the web page where the interaction occurred, as recorded by an `ExperienceEvent`",
```

Comment:
This description is very hazy to me, and easily confused with the description for `webPage`. It would probably make sense to move both into a separate `*.description.md`

The way I see it, we have two things that are relevant here:
- the web page where an interaction is happening
- the hyperlink that has lead to the page where the interaction is recorded

The description as it stands, says that `webpage` is the page where the interaction happened. And `weblink` is about this, too.

Reply:
@trieloff - No, you understand this incorrectly. There are two kinds of web events, a PageView(a full new loading of a web page) and a Page Event which is an action happening inside a page that has already had a PageView. The latter does not increment a page count, but will be used in single page hits to indicate that the user really interacted with the page and thus take it out of a bounce count.
@kstreeter FYI

Reply:
The Web Link is normally used for the single page app sort of case. So analogous to s.tl() in AppMeasurement. It is best to think of the first class nature of this being a Page Event. If this were a pure PageView then we may not have this sort of referrer information, if we do then good it can appear here. But the primary case is where there is a page loaded, and a local link (anchor) event occurs that indicates the end user drilling into some aspect of the page. Thus a "link-click" event would always have its primary data for the link being executed in this "Web Link"

Reply:
Ah. That makes things a bit clearer. Let me replay that, so that we can transfer the emerging clarity into the schema:

- we have two types of interactions: an entire page is loaded vs. an interaction happens on a page that does not trigger a page load
- for each type of interaction we want to capture a couple of properties including where (URL, page name, link name)

The names we use are:

|What happened|Adobe Analytics|This PR|My Proposal|
|----------------|----------------|--------|------------|
|A page was loaded| Page View | `webinfo`| `webpageview` |
|An interaction on a page was recorded| Page Event | `weblink` | `webinteraction` |

@lrosenthol please take a look: I don't want to add further confusion

@cdegroot-adobe if that looks alright, let's rename the two buckets, and give them proper descriptions (too much is buried here in these review threads)